samples/sensor/accel_polling/sample.yaml

```diff
@@ -0,0 +1,50 @@
+sample:
+  name: Accelerometer polling sample
+tests:
+  sample.sensor.accel_polling:
+    harness: console
+    tags: sensors
+    filter: dt_alias_exists("accel0")
+    platform_exclude: sensortile_box thingy53_nrf5340_cpuapp lpcxpresso55s69_cpu1
+    harness_config:
+      type: one_line
+      regex:
+        - "^\\s*[0-9A-Za-z_,+-.]*@[0-9A-Fa-f]* \\[m\/s\\^2\\]:    \\(\\s*-?[0-9\\.]*,\\s*-?[0-9\\.]*,\\s*-?[0-9\\.]*\\)$"
+    integration_platforms:
+      - cc1352r_sensortag               # adxl362
+      - blueclover_plt_demo_v2_nrf52832 # bmi270
+      - frdm_kl25z                      # mma8451q
+      - lpcxpresso55s28                 # mma8652fc
+      - bbc_microbit                    # mmc8653fc
+      - frdm_k64f                       # fxos8700
+      - sparkfun_thing_plus_nrf9160     # lis2dh
+      - thingy52_nrf52832               # lis2dh12
+      - stm32f411e_disco                # lsm303agr_accel
+      - stm32f3_disco                   # lsm303dlhc_accel
+      - bl5340_dvk                      # lis3dh
+      - b_l4s5i_iot01a                  # lsm6dsl
+
+  sample.sensor.accel_polling.sensortile_box:
+    harness: console
+    tags: sensors
+    platform_allow: sensortile_box
+    harness_config:
+      type: multi_line
+      regex:
+        - "^\\s*lis2dw12@0 \\[m\/s\\^2\\]:    \\(\\s*-?[0-9\\.]*,\\s*-?[0-9\\.]*,\\s*-?[0-9\\.]*\\)$"
+        - "^\\s*lsm6dso@1 \\[m\/s\\^2\\]:    \\(\\s*-?[0-9\\.]*,\\s*-?[0-9\\.]*,\\s*-?[0-9\\.]*\\)$"
+        - "^\\s*iis3dhhc@2 \\[m\/s\\^2\\]:    \\(\\s*-?[0-9\\.]*,\\s*-?[0-9\\.]*,\\s*-?[0-9\\.]*\\)$"
+    integration_platforms:
+      - sensortile_box                  # lis2dw12, lsm6dso, iisdhhc
```

Comment:
IMO this overcomplicates the sample.yaml, the generic entry should be sufficient for all cases even if not perfect, in the end we're validating that something is printed...

Reply:
Removing board specific cases.